p2p/net/swarm/addr/addr.go

```diff
@@ -42,10 +42,14 @@ func init() {
 
 // FilterAddrs is a filter that removes certain addresses, according to filter.
```

Comment:
"according to *a list* of filters"

"if *all* of the filters return true, the address is kept"